lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
```

Comment:
I restored the `workingDirectory` parameter. When I tried two days ago the Analyzer didn't hint about the use of deprecated named arguments, this is why it didn't seem to add value to keep them around, but now I get hints from the analyzer and then it's definitely worth keeping them around a while.